enhancements/update/cluster-profiles.md

```diff
@@ -82,8 +82,8 @@ out of the scope of this design.
 CLUSTER_PROFILE=[identifier]
 ```
 This environment variable would have to be specified in the CVO deployment. When
-no `CLUSTER_PROFILE=[identifier]` variable is specified, the `default` cluster profile
-is in effect.
+no `CLUSTER_PROFILE=[identifier]` variable is specified, the `self-managed-highly-available` 
+cluster profile is in effect.
```

Comment:
`self-managed-highly-available` seems like a bad name for the "default" profile unless "who manages it" and "whether it is highly available" are the only axes on which future profiles are allowed to differ from the default.

Like, if you weren't peeking ahead at future plans, why would you call out `self-managed` and `highly-available` as the defining attributes of the default OCP experience, but not mention `live-upgradeable` or `rhcos-control-plane` or various other things?

Reply:
These are the known axes, e.g. ROKS moves some control plane components off-cluster (so not self-managed) and CRC is dev-oriented, low resource (so not HA).  If we invent more axes later, maybe we'll wish we had been more specific, but 🤷🔮

Reply:
> If we invent more axes later, maybe we'll wish we had been more specific

Yeah, I meant we should be _less_ specific; just call it `default`, rather than trying to guess now all of the ways it's going to be different from other future profiles.